crates/libcontainer/src/syscall/linux.rs

```diff
@@ -559,6 +559,24 @@ impl Syscall for LinuxSyscall {
         }?;
         Ok(())
     }
+
+    fn set_io_priority(&self, class: i64, priority: i64) -> Result<()> {
+        let ioprio_who_pgrp = 1;
+        let iop = (class << 13) | priority;
+        match unsafe {
+            libc::syscall(
+                libc::SYS_ioprio_set,
+                ioprio_who_pgrp as libc::c_int,
+                0,
+                iop as libc::c_ulong,
+            )
+        } {
+            0 => Ok(()),
+            -1 => Err(nix::Error::last()),
+            _ => Err(nix::Error::UnknownErrno),
+        }?;
+        Ok(())
```

Comment:
Is there any other way of doing this than raw syscall? Does nix crate have anything related to this?

Reply:
in `nix` crate not find, but find [`ioprio`](https://docs.rs/ioprio/latest/ioprio/) crate.

Reply:
There is no libc wrappers yet, so syscall it is. In general, for us, I prefer not to drag another crate into the picture. We usually use the syscall in a very specific way, instead of covering all the use cases that a syscall may provide. To me this is OK.

Reply:
Yeah,  I was wondering if nix has it ; but if it doesn't, no point in adding a crate just for that in this case :+1: